go/internal/cbor/deterministic_test.go

```diff
@@ -188,6 +188,54 @@ func TestByteStringIsNotDeterministic(t *testing.T) {
 	})
 }
 
+func TestMapIsDeterministic(t *testing.T) {
+	hello := multiappend([]byte{0b01100101}, []byte("hello"))
```

Comment:
Binary-encoded CBOR headers like `[]byte{0b01100101}` are hard to read. Can you create a helper function that constructs a CBOR header, for readability?

Reply:
Doing it in a separate CL sounds good.